setup.cfg

```diff
@@ -1,2 +0,0 @@
-[pydocstyle]
```

Comment:
This file was only used for config, which was merged into `.flake8` and thus can be removed.